django/middleware/csrf.py

```diff
@@ -106,10 +106,7 @@ def rotate_token(request):
     Change the CSRF token in use for a request - should be done on login
     for security purposes.
     """
-    request.META.update({
-        "CSRF_COOKIE_USED": True,
-        "CSRF_COOKIE": _get_new_csrf_token(),
-    })
+    request.META["CSRF_COOKIE"] = _get_new_csrf_token()
```

Comment:
I would remove the last commit, this is not necessary for this ticket. Moreover, as far as I'm aware ticket-32916 suggests to do the opposite and remove rather `csrf_cookie_needs_reset`.

Reply:
Okay, I'll remove it. I added this commit because it's related to the PR: `rotate_token()` doesn't imply `"CSRF_COOKIE_USED"` but does imply needing a reset. (The ticket suggests using a `"CSRF_COOKIE_NEEDS_RESET"` key rather than `"CSRF_COOKIE_USED"` because it's the former that is named appropriately for all cases.)

Reply:
Right, I missed that the ticket suggest a different name. I would still leave it for ticket-32916.

Reply:
Okay, I removed the commit. (If [ticket-32916](https://code.djangoproject.com/ticket/32916) isn't accepted, I still think the commit should be considered separately for the reason I stated above -- that only cookie needs reset is true in this case.)